core/src/allocation_init.jl

```diff
@@ -856,6 +856,9 @@ function AllocationModel(
 
     # The JuMP.jl allocation problem
     problem = allocation_problem(config, p, capacity, allocation_network_id)
+    if (config.allocation.objective_type != "linear_absolute")
+        error("Type of object function is not supported")
+    end
```

Comment:
```suggestion
    if config.allocation.objective_type != "linear_absolute"
        error("Type of object function is not supported")
    end
```

We typically don't use brackets here if not needed.